### ISSUE TITLE
Allow ToggleVariant hotkey to by available for all mods

### ISSUE THREAD

**Issue by Porenutak** [OpenRA/OpenRA#20337]
Right now **ToggleVariant** hotkey for building rotation is not listed in Hotkeys GUI menu. You must add it manually in to hotkeys.yaml with text editor. This is not comfortable for common players how often miss this feature. 
Even if default mods dont have any buildingVariants its good to have this hotkey available. Example: in D2k we have Refinery variants added through map mods.

**Comment by PunkPun** [comment]
I'm not sure what is the best way to support that. 

It doesn't make sense to display it in the hotkeys menu by default if the mod doesn't use it. Perhaps add an unused hotkey section?

**Comment by dragunoff** [comment]
I think a section for unused hotkeys doesn't make much sense - what's the purpose of an unused key? And that won't be entirely true because it can become used in modded maps :thinking: Maybe we should just add it to the list but without binding it to anything be default.

**Comment by abcdefg30** [comment]
That seems like something which can be explained in a map briefing. Displaying unused hotkeys doesn't sound great.

**Comment by Porenutak** [comment]
I agree. There should be default bind key. We can use some key from keyboard middle section as this hotkey would not be used often. You can notice players which building can be rotate in briefing. We already do that in D2k tournament maps also with tutorial how to add hotkey manually with text editor. But lot of players was confused or complain why they must edit this hotkey in text editor and not in shortcut menu. That's why I open this issue.
